src/variables.scss

```diff
@@ -20,4 +20,8 @@ under the License.
 
 $primaryColor: #e33e6b;
 $accentColor: #3e6be3;
-
+$top-menu-height: 40px;
+$left-menu-width: 48px;
+$menu-border: solid 1px rgba(0, 0, 0, 0.12);
```

Comment:
It is unused. Could be `1px solid #dadada` to be used for the `::ng-deep .top-menu border-bottom`